jason-jasonmeyers/hw_5/7-jquery-ajax-exercise/js/main.js

```diff
@@ -0,0 +1,44 @@
+/*
+
+1. Sign up for openweathermap.org and generate an API key.
+2. User either $.ajax or $.get to pull weather current data .
+   for San Francisco (hint: http://api.openweathermap.org/data/2.5/weather?q=...).
+3. Print the temperature in the console. Note that the temperature will be in Kelvin.
+4. Incorporate the following code into your app to log a Fahrenheit value for the temperature to the console
+   (Fahrenheit temperature is stored in the degFInt variable):
+   var degF = (temp - 273.15) * 1.8 + 32;
+   var degFInt = Math.floor(degF);
+5. Use DOM manipulation to add the returned temperature to the span element with the id "temp".
+6. Sign up for developer.weatherunlocked.com and obtain your application ID and application key.
+7. Repeat the above steps to obtain the current temperature from this data source, convert it to Fahrenheit,
+   log it to the console, and then add it to the DOM (uncomment out the 3 lines indicated in index.html for this step,
+   and add your temperature value to the span elmeent with the id "temp2").
+8. Refactor your code as necessary to separate the code for DOM manipulation from the code for sending an HTTP request.
+BONUS: Implement a form that prompts users for a city and state and returns data for the location they specify
+   (Uncomment out the existing form code in index.html as marked).
+
+*/
+
+'use strict';
+
+var getLocation = document.getElementById('location');
+var getLocationData = encodeURIComponent(getLocation.innerHTML);
+var getTemp = document.getElementById('temp');
+
+function returnF(temp){
+  var degF = (temp - 273.15) * 1.8 + 32;
+  var degFInt = Math.floor(degF);
+  return degFInt;
+}
+
+var weatherUrl = "http://api.openweathermap.org/data/2.5/";
+var queryType = "weather?";
+var weatherLocation = "q=" + getLocationData;
+var apiIdent = "&appid=";
+var apiKey = "02c537feb8ec33139ffffbe75928b8b2";
```

Comment:
Be mindful of pushing up API keys in the future. Best case scenario they'll be disabled by the API provider, and worst case could result in losing real dollars:
https://wptavern.com/ryan-hellyers-aws-nightmare-leaked-access-keys-result-in-a-6000-bill-overnight